code/modules/mob/inventory.dm

```diff
@@ -280,9 +280,15 @@
  * If the item can be dropped, it will be forceMove()'d to the ground and the turf's Entered() will be called.
 */
 /mob/proc/dropItemToGround(obj/item/I, force = FALSE, silent = FALSE, invdrop = TRUE)
+	if (isnull(I))
+		return FALSE
```

Comment:
this might be the thing breaking snails?

Reply:
```
	if(!I) //If there's nothing to drop, the drop is automatically succesfull. If(unEquip) should generally be used to check for TRAIT_NODROP.
		return TRUE

```

Reply:
oh wait but it immediately reutns true ugh

Reply:
it was